crest/Assets/Crest/Crest/Shaders/Underwater/UnderwaterCurtain.shader

```diff
@@ -212,7 +212,9 @@ Shader "Crest/Underwater Curtain"
 #if _CAUSTICS_ON
 				if (sceneZ01 != 0.0)
 				{
-					ApplyCaustics(view, lightDir, sceneZ, _Normals, true, sceneColour, cascadeData0, cascadeData1);
+					float3 camForward = mul((float3x3)unity_CameraToWorld, float3(0.0, 0.0, 1.0));
```

Comment:
There a way to pull out the forward vector directly which avoids a matrix mul. May be worth it as this code runs over many pixels. Can never remember which but it's either mat._m20_m21_m22 or mat._m02_m12_m22 or mat[2]

Reply:
Thanks. Changed all of them to _m02_m12_m22 which worked.